sync/class.jetpack-sync-actions.php

```diff
@@ -124,6 +124,8 @@ static function send_data( $data, $codec_name, $sent_timestamp, $queue_id ) {
 			'codec'     => $codec_name, // send the name of the codec used to encode the data
 			'timestamp' => $sent_timestamp, // send current server time so we can compensate for clock differences
 			'queue'     => $queue_id, // sync or full_sync
+			'home'      => get_option( 'home' ), // Send home url option to check for Identity Crisis server-side
```

Comment:
Should we send both the option value and the function results `get_home_url`, `get_site_url`?